foundation-models/openai/pom.xml

```diff
@@ -149,6 +154,12 @@
       <artifactId>javaparser-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.ai</groupId>
+      <artifactId>spring-ai-client-chat</artifactId>
+      <scope>test</scope>
```

Comment:
(Minor/Question)

Do we not have to put this to **compile** scope (+ optional)?

Reply:
Nope, used for hat memory testing